File: program1_4.java
//Accept one number and check is it devisible by 3 and 5 or not

import java.util.*;

class DivideX
{
    int iNo;

    public DivideX(int iValue)
    {
        this.iNo=iValue;
    }

    public boolean Divisible()
    {
        boolean bFlag=false;

        if((iNo%3)==0 && (iNo%5)==0)
        {
            bFlag=true;
        }
        return bFlag;
    }
}
class program1_4
{
    public static void main(String A[])
    {
        Scanner sobj = new Scanner(System.in);
        boolean bRet=false;

        System.out.println("Enter Number:");
        int iValue= sobj.nextInt();

        DivideX dobj =new DivideX(iValue);
        
        bRet=dobj.Divisible();

        if(bRet==true)
        {
            System.out.println(iValue+" Number is divisible by 3 & 5");
        }
        else
        {
             System.out.println(iValue+" Number is NOT divisible by 3 & 5");
        }



    }
}

File: Program1.4.c
///////////////////////////////////////////////////////////////////////
//  Problem statement:Accept one number and check whether is it divisible by 5 or not
//  Input:
//  Output:
//////////////////////////////////////////////////////////////////////

#include<stdio.h>
#include<stdbool.h>

//Getting error when use typedef
#define TRUE 1
#define FALSE 2

bool Check(int iNo)
{
    if((iNo%5)==0)
    {
        return true ;
    }
    else
    {
        return false ;
    }
}
int main()
{
    int iValue=0;
    bool bRet=false;

    printf("Enter Number:\n");
    scanf("%d",&iValue);

    bRet=Check(iValue);

    if(bRet==true)
    {
        printf("Divisible by 5\n");
    }
    else
    {
        printf("Not Divisible by 5\n");
    }


    return 0;

}